### PR TITLE
sample pipeline notebook

### PR DESCRIPTION
**Pull request by animeshsingh**
Sample pipeline notebook for KFServing, to be be followed with more complex ones showing interaction with  other kubeflow components

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/476)

### CODE REVIEW

File: docs/samples/pipelines/kfs-pipeline.ipynb [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    "  name='kfserving pipeline',\n",
+    "  description='A pipeline for kfserving.'\n",
+    ")\n",
+    "def kfservingPipeline(\n",
+    "    action = 'update',\n",
+    "    model_name='tf-sample',\n",
+    "    default_model_uri='gs://kfserving-samples/models/tensorflow/flowers',\n",
+    "    canary_model_uri='gs://kfserving-samples/models/tensorflow/flowers-2',\n",
+    "    canary_model_traffic_percentage='10',\n",
+    "    namespace='kubeflow',\n",
```
Comment (ellis-bigelow) [inline docs/samples/pipelines/kfs-pipeline.ipynb]: Don't deploy to the kubeflow namespace. This is a system namespace.

Comment (animeshsingh) [reply docs/samples/pipelines/kfs-pipeline.ipynb]: Done @ellis-bigelow

**Comment by jinchihe** [comment]
@animeshsingh Maybe need to rebase to resolve CI testing problem? since the downgrade istio and knative in CI tests. Thanks.

**Pull request merged**